apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations-v2/[evaluationUuid]/editor/_components/EvaluationEditor/index.tsx

```diff
@@ -153,7 +150,12 @@ export function EvaluationEditor({
         secondPane={
           <SplitPane.Pane>
             <div className='flex-1 relative max-h-full pr-6'>
-              <Playground metadata={metadata!} />
+              <Playground
+                commit={commit}
+                document={document}
+                evaluation={evaluation}
+                metadata={metadata!}
+              />
             </div>
           </SplitPane.Pane>
         }
```

Comment:
this is probably inspired from v1 evaluation editor which has horrible performance optimizastions compared to the document editor/playground, in case you wanna revisit this you can take inspiration from there.

Reply:
No, is inspired in document playground